.carve/ignore

```diff
@@ -0,0 +1,216 @@
+;; Used by shadow-cljs
+electron.core/main
+electron.core/start
+electron.core/stop
+;; repl fn
+electron.search/query
+frontend.blob/blob
+frontend.components.sidebar/nav-item
+frontend.components.svg/arrow-left
```

Comment:
There are a lot of unused fns from the frontend.components.svg namespace? Should I remove these fns or ignore the namespace?